Getting-Started.md

```diff
@@ -709,4 +753,54 @@ Kibiter then automatically generates a query as shown below.
 
 > :warning: **If you change the structure or delete it** you won't see the menu.
 
-> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
\ No newline at end of file
+> **Note**: the `<id_of_the_dashboard>` must be the identifier that Kibana/Kibiter put to a saved dashboard.
+
+#### Dump the index mapping/data with elasticdump [&uarr;](#dump-data/mapping-)
+
+Indexes and mappings can be dumped (or uploaded) using [elasticdump](https://github.com/taskrabbit/elasticsearch-dump). The commands to install elasticdump, dump an index and its index pattern to a file are shown below.
+
+**elasticdump installation**
+
+```
+sudo apt install npm
+sudo -i
+npm install elasticdump -g
+```
+
+CTRL+D to logout from root user
+
+***
+
+> Use the 'type' argument to either fetch data or mapping.
+
+
+**Dump the index data**
+
+```
+elasticdump --input=http://localhost:9200/git_chaoss/ --output=git_data.json --type=data
+```
+
+![elasticdump_data](https://user-images.githubusercontent.com/53489999/77806813-d4cacb00-70ab-11ea-9a58-dcb3c7715e9d.png)
+
+**Dump the index mapping**
+
+```
+elasticdump --input=http://localhost:9200/git_chaoss/ --output=git_mapping.json --type=mapping
+```
+
+![elasticdump_mapping](https://user-images.githubusercontent.com/53489999/77806637-40f8ff00-70ab-11ea-8fe5-766fbde93fcf.png)
+
+
+OUTPUT FILES 
```

Comment:
Please remove this part (lines 794 - 804) since it doesn't provide additional info to dump the indexes/mappings.